### DISCUSSION TITLE
Adding support for alarm panel card

### DISCUSSION THREAD

**Discussion by smarthomefamilyverrips** [Blackymas/NSPanel_HA_Blueprint#227] [Ideas]
Would it be possible to add the alarmo code panel card?

**Comment by ghost** [comment]
Hi
There have been a few requests for this, there are these 2 open issues which have been accepted as feature requests so it should be implemented in the future
 #104 
#66

**Reply by sqalg** [reply]
> Would it be possible to add the alarmo code panel card?

I'm interested to.
Do we have a chance to have it quickly?
thanks
Great job for all you already done

**Comment by edwardtfn** [comment]
I need help with the translations for this:

https://github.com/Blackymas/NSPanel_HA_Blueprint/discussions/940

**Comment by edwardtfn** [comment]
I'm closing this discussion as we have released v4.0 (#1057).
Please add a feature request for the missing things.

And thanks a lot for your support.
